services/IoT/applications/libraries/python.md

```diff
@@ -860,21 +855,21 @@ The response includes a list of log entries, which contain log messages and time
 
 The historical event retrieval operations that are described in this section can be used to view events from all devices, view events from a device type, or to view events for a specific device.
 
-For information about the query parameters, the request & response model and HTTP status code, see the Historical event retrieval section of the [{{site.data.keyword.iot_short_notm}} API](https://docs.internetofthings.ibmcloud.com/swagger/v0002.html).
+For information about the query parameters, the request & response model, and HTTP status codes, see the Historical event retrieval section of the [{{site.data.keyword.iot_short_notm}} API](https://docs.internetofthings.ibmcloud.com/swagger/v0002.html).
 
 
 ### Viewing events from all devices
 
-Use the *getHistoricalEvents()* method to view events across all devices registered to the organization.
+Use the *getHistoricalEvents()* method to view events from all devices that are registered to the organization.
 
 ```
 
     print("Historical Events = ", apiCli.getHistoricalEvents())
 ```
 
-The response of the *getHistoricalEvents()* method contains some parameters and the application needs to retrieve the JSON element *events* from the response to get the array of events returned. Other parameters in the response are required to make further call, for example, the *_bookmark* element can be used to page through results. Issue the first request without specifying a bookmark, then take the bookmark returned in the response and provide it on the request for the next page. Repeat until the end of the result set indicated by the absence of a bookmark. Each request must use exactly the same values for the other parameters, otherwise the results are undefined.
+The response of the *getHistoricalEvents()* method contains some parameters, and the application needs to retrieve the JSON element *events* from the response to get the array of events returned. Other parameters in the response are required to make more calls, for example, the *_bookmark* element can be used to page through results. Issue the first request without specifying a bookmark, then take the bookmark that is returned in the response and provide it on the request for the next page. Repeat until the end of the result set, which is indicated by the absence of a bookmark. Each request must use exactly the same values for the other parameters, otherwise the results are undefined.
```

Comment:
line 870, 893, 906
"and the application needs to retrieve the JSON element *events* from the response to get the array of events returned."
does this mean "to return the array of devices" or "to get the array of devices that is returned"